hl/src/H5LTanalyze.c

```diff
@@ -2413,11 +2413,7 @@ yy_scan_buffer(char *base, yy_size_t size)
  *       yy_scan_bytes() instead.
  */
 YY_BUFFER_STATE
-yy_scan_string(const char *yystr)
-{
-
-    return yy_scan_bytes(yystr, (int)strlen(yystr));
-}
+yy_scan_string(const char *yystr) { return yy_scan_bytes(yystr, (int)strlen(yystr)); }
 
```

Comment:
Is this due to running the formatter over this file?

Reply:
Yes, I thought I had this file excluded.

Reply:
It is excluded in the yaml, just not in the bin scripts. I'll fix that.

Reply:
I can't seem to find a previous version, but it shouldn't matter much as it should be the same after formatting.